### DISCUSSION TITLE
Can vanilla-extract export plain CSS files (from multiple entries)?

### DISCUSSION THREAD

**Discussion by ngdangtu-vn** [vanilla-extract-css/vanilla-extract#1179] [Q&A]
With SASS I usually use this structure:

```
- main.scss
- _reset.scss
- page/front.scss
- page/e404.scss
- component/_card.scss
```

With any files that don't start with `_` is entry files. And the result of compiling is 3 CSS files. Can I do the same thing on vanilla-extract? Also, my CSS will be used by HTML generated by PHP so there is no way I can guess all the hash class, can I disable it as well?

**Comment by ngdangtu-vn** [comment]
I guess we cannot.

**Reply by askoufis** [reply]
What lead you to this conclusion? Generally speaking, your bundler is what controls how CSS files are output. Also, [you can now customize classname identifiers](https://vanilla-extract.style/documentation/integrations/rollup/#identifiers), which may help if you want to have more control over hashes.
